analyzers/its/expected/Automapper/AutoMapper--net461-S3900.json

```diff
@@ -405,6 +405,19 @@
 },
 {
 "id":  "S3900",
+"message":  "Refactor this method to add validation of parameter 'memberNames' before using it.",
+"location":  {
+"uri":  "sources\Automapper\src\AutoMapper\Configuration\MappingExpression.cs",
+"region":  {
+"startLine":  37,
```

Comment:
Looks good
https://github.com/SonarSource/sonar-dotnet/blob/3420f6cc2ab44de7fba95030d2770633430ec625/analyzers/its/sources/AutoMapper/src/AutoMapper/Configuration/MappingExpression.cs#L34-L37

Reply:
I've added back the deleted line, and now it's gone. There was a PR earlier regarding Flow Capture which would have fixed it, but we decided to cut branches, so this will remain a FN.